sigopt_sklearn/search.py

```diff
@@ -274,7 +276,8 @@ def _fit(self, X, y, parameter_iterable=None):
         pre_dispatch = self.pre_dispatch
 
         # setup SigOpt experiment and run optimization
-        self._create_sigopt_exp(self.sigopt_connection)
+        if self.experiment is None:
+            self.experiment = self._create_sigopt_exp(self.sigopt_connection)
```

Comment:
Made public in https://github.com/sigopt/sigopt_sklearn/pull/7/commits/9bd4d5757c7f53542b72ccc02c2653697d730932. Everything is still automatic if the user does nothing, and this can be used to create an experiment in exactly the way one would expect otherwise.